lms/views/lti/jwk.py

```diff
@@ -0,0 +1,8 @@
+from pyramid.view import view_config
+
+from lms.services import RSAKeyService
+
+
+@view_config(route_name="lti.jwks", request_method="GET", renderer="json")
+def jwks(request):
```

Comment:
We should have a doc string here describing what purpose this end-point serves.

In general, we should document everything, rather than it being an exception.